src/modules/users/users.service.ts

```diff
@@ -45,6 +45,18 @@ export class UsersService extends BaseRepositoryService<User> {
       .getMany();
   }
 
+  getUserByType(whereOptions: object, type: string): Promise<User> {
```

Comment:
Why type is `object ` ?

Reply:
Because we can pass many different parameters each time and it is really redundant to create type for every modal for such cases